packages/block-editor/src/components/block-breadcrumb/README.md

```diff
@@ -0,0 +1,31 @@
+# Block Breadcrumb
+
+The block breadcrumb trail displays the hierarchy of the current block selection as a breadcrumb. It also allows, using this hierarchy, to navigate to the parent elements of the current block selection. It is located at the very bottom of the editor interface.
+
+![Button block breadcrumb](https://make.wordpress.org/core/files/2020/08/gutenberg-button-block-breadcrumb.png)
+
+![Image in column block breadcrumb](https://make.wordpress.org/core/files/2020/08/gutenberg-image-in-column-block-breadcrumb.png)
+
+## Table of contents
+
+1. [Development guidelines](#development-guidelines)
+2. [Related components](#related-components)
+
+
+## Development guidelines
+
+### Usage
+
+Renders a block breadcrumb with default style.
+
+```jsx
+import { BlockBreadcrumb } from '@wordpress/block-editor';
+
+<div className="add-your-class-name">
+    <BlockBreadcrumb />
+</div>
```

Comment:
This is bad code, it needs to be wrapper inside a function (component), we can't just use JSX top level like that.

Reply:
something like 
`const MyBreadcrumb = () => <BlockBreadcrumb />;` I guess.

I think I can also remove the div.